notebooks/01a-cosmos_tutorial.ipynb

```diff
@@ -68,9 +68,7 @@
    "id": "975ddc32",
```

Comment:
<p>I noticed the names of the files are a bit confusing <code>_fits.fits</code>,is there another name we can use to make it clear the information that each file contains?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSSTDESC/BlendingToolKit/pull/320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
maybe a different suffix we can use for both catalog files? I think we can change all filenames accordingly if there is a better name you can think of

Reply:
Yes I used those names to follow the convention of the original COSMOS dataset, i.e., `catalog.fits` for the catalog containing information for real galaxies and `catalog_fits.fits` for the catalog containing the information for parametric galaxies. The thing is that galsim automatically looks for files named with this convention, so I do not think we can set other names unless we change galsim...
When making our extended galsim COSMOS dataset, I also wondered in which catalog I should put the EL-COSMOS HSC multiband magnitudes (both options are possible because we merge both catalogs in the BTK catalog class) and I think the second one makes sense because those magnitudes come from fits, but I may put them in the first catalog (along with the original HST magnitude of galaxies) if we think it makes more sense.